svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/[issueSearchType=issue_search_type]/+page.server.ts

```diff
@@ -94,9 +109,25 @@ export const load: PageServerLoad = async ({ fetch, params, url: { searchParams,
     }
   );
 
+  const labelParameterDictionaryMilestones = openMilestones.reduce((dict, x) => {
+    dict[x.title] = buildFilterParameter(
+      SEARCH_QUERY_PARAMETER_QUALIFIER.MILESTONE,
+      `"${x.title}"`
+    );
+    return dict;
+  }, {} as Record<string, string>);
+  const milestoneFilters = buildNavigationFilterOptions(
+    href,
+    searchQuery,
+    labelParameterDictionaryMilestones,
+    (label) => label,
+    true
+  );
+  console.log(labelParameterDictionaryMilestones);
   return {
     issues,
     sortFilters,
     stateFilters,
+    milestoneFilters,
```

Comment:
There is a lot going on in this `load()` that looks like it could probably be simplified, but we can address this with refactoring

Reply:
No, this is fine.